lib/puppet/type/group.rb

```diff
@@ -80,7 +80,9 @@ def sync
 
     newproperty(:members, :array_matching => :all, :required_features => :manages_members) do
       desc "The members of the group. For directory services where group
```

Comment:
The fragment is "For directory services where group membership is stored in the group objects, not the users." It's a bit outside the context of this particular ticket, but if you have a good suggestion, I can make the change.